esp-hal/CHANGELOG.md

```diff
@@ -45,17 +45,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed the parameters of `Spi::with_pins` to no longer be optional (#2133)
 - Renamed `DummyPin` to `NoPin` and removed all internal logic from it. (#2133)
 - The `NO_PIN` constant has been removed. (#2133)
+- The various `dma_buffer` macros now return the buffers as `&mut [u8]` (#2179)
```

Comment:
Is this changelog still correct? It looks to me like we're returning `&mut [u8; SIZE]`?

Reply:
yup Dominic also noticed, I rebased away the commit that removed it

Reply:
Dominic beat me to it :D